frontend/src/pages/Students.vue

```diff
@@ -0,0 +1,326 @@
+                  <template v-slot="props">
+                      {{ props.row.PhoneNumber }}
+                  </template>
+                  </b-table-column>
+              </template>
+              
+          </b-table>
+      </section>
+    </div>
+  </Page>      
+</template>
+
+
+<script>
+
+// Placeholder for API - Note that this will only work after the FullName method has been merged in the backend//
+const API_URL = "http://localhost:3001/api/students";
+
+import PageHeader from '../components/PageHeader.vue'
+import Page from '../components/Page.vue'
+
+
+export default {
+        data() {
+            return {
+                data: [
+                    //Placeholder data if you don't want to run the backend
+                    // { 'StudentID': 12345, 'FullName': 'Jesse Simmons', 'created_at': '2020-10-24T06:18:24.738Z', 'StatusID': 1,'PhoneNumber': '91233217' },
+                    // { 'StudentID': 23456, 'FullName': 'John Jacobs', 'created_at': '2020-10-25T06:18:24.738Z', 'StatusID': 2, 'PhoneNumber': '91312231' },
+                    // { 'StudentID': 31232, 'FullName': 'Tina Gilbert', 'created_at': '2020-10-26T06:18:24.738Z', 'StatusID':  3, 'PhoneNumber': '81234102'},
+                    // { 'StudentID': 41231, 'FullName': 'Clarence Flores', 'created_at': '2020-10-26T06:18:24.738Z', 'StatusID': 4,  'PhoneNumber': '93141234' },
+                    // { 'StudentID': 53212, 'FullName': 'Anne Lee', 'created_at': '2020-10-27T06:18:24.738Z', 'StatusID': 1,  'PhoneNumber': '81230532' },
+                ],
+                selected: null,
+                sortIcon: 'arrow-up',
+                sortIconSize: 'is-small',
+                sortDirection: 'asc',
+                columns: [
+                    {
+                        field: 'created_at',
+                        label: 'Date Joined',
+                        searchable: true,
+                    },
+                    {
+                        field: 'StudentID',
+                        label: 'Student ID',
+                        searchable: true,
+                    },
+                    {
+                        field: 'FullName',
+                        label: 'Student Name',
+                        searchable: true,
+                    },                               
+                    {
+                        field: 'StatusID',
+                        label: 'Status',
+                        searchable: true,
+                    },
+                    {
+                        field: 'PhoneNumber',
+                        label: 'Phone Number',
+                        searchable: true,
+                    }
+                ]
+            }
+        },
+        computed:{
+          tableData(){
+            return this.data.map(student => {
+              return {
+                StudentID: `${student.StudentID}`,
+                FullName: `${student.FullName}`,
+                created_at: `${student.created_at}`,
```

Comment:
```suggestion
                CreatedAt: `${student.created_at}`,
```
Let's make this consistent with the other column names. After you change this, don't forget to update `created_at` anywhere where you referenced it (e.g. in the columns on Line 190, Line 34).